cron/workspaceHostTransitions.js

```diff
@@ -106,7 +107,7 @@ async function checkHealth() {
       healthy = false;
     } else {
       try {
-        await request({ uri: url, resolveWithFullResponse: true });
+        await fetch(url, { signal: AbortSignal.timeout(30_000) });
```

Comment:
TIL about `AbortControllers` -- neat!

Reply:
Just to make sure I'm understanding, this is using `fetch()` from `node-fetch`, not the built-in `fetch()` (currently experimental?), but we are using the built-in `AbortSignal` (added in 16.14, I think)? I assume this all plays together nicely?

Reply:
Yeah, `node-fetch` has supported `signal` for a long time, and we're running Node 16.19, so we should be good!